.prettierignore

```diff
@@ -1,3 +1,2 @@
 build
 node_modules
-charting_library
```

Comment:
What is this change and why do we need it?

Reply:
There is no charting_library folder in the repo. 1, 2 - it is the folders where is disabled Prettier.